app/models/user.rb

```diff
@@ -18,4 +18,12 @@ class User < ApplicationRecord
             uniqueness: true,
             length: { in: 3..20,
                       wrong_length: 'must be between 3 and 20 characters' }
+
+  def attendee_of?(event)
+    EventAttendee.find_by('attendee_id=? AND attended_event_id=?', id, event.id).present?
```

Comment:
this can be re-written in a more railsy and performant way. Note that `#exists?` checks if the query returns something at the SQL level, but `#present?` needs to execute the SQL query, parse the query result and load it in memory just to check if it isn't blank.

Also remember to give more priority to writing your queries using the active record interface instead of writing SQL

```suggestion
    event_attendees.where(attended_event_id: event_id).exists?
```

Reply:
done!